items.md

```diff
@@ -237,6 +238,58 @@ It returns the bitstreams within this item. See the bitstream endpoint for more
 The supported parameters are:
 * page, size [see pagination](README.md#Pagination)
 
+**POST /api/core/items/<:uuid>/bitstreams**
+
+Example: <https://dspace7.4science.it/dspace-spring-rest/#https://dspace7.4science.it/dspace-spring-rest/api/core/items/1911e8a4-6939-490c-b58b-a5d70f8d91fb/bitstreams>
+
+Curl example:
+```
+curl 'https://dspace7.4science.cloud/dspace-spring-rest/api/core/items/2f4ec582-109e-4952-a94a-b7d7615a8c69/bitstreams' \
+ -XPOST -H 'Content-Type: multipart/form-data' \
+ -H 'Authorization: Bearer eyJhbGciOiJI...' \
+ -F "file=@Downloads/test.html" \
+ -F 'properties={ "name": "test.html", "metadata": { "dc.description": [ { "value": "example file", "language": null, "authority": null, "confidence": -1, "place": 0 } ]}, "bundleName": "ORIGINAL" };type=application/json'
+```
+
+* The item is determined using the ID in the URL
+* The file is uploaded using multipart/form-data
+* The metadata of the bitstream is included as a json property, and with layout, it looks like:
+```json
+{
+  "name": "test.html",
+  "metadata": {
+    "dc.description": [
+      {
+        "value": "example file",
+        "language": null,
+        "authority": null,
+        "confidence": -1,
+        "place": 0
+      }
+    ]
+  },
+  "bundleName": "ORIGINAL"
+}
+```
+The bitstream metadata can contain:
+* The filename to be stored (optional)
+* metadata for the bitstream (optional)
+* bundleName (mandatory)
+* sizeBytes (optional, can be verified on the server to verify successful transfer)
+* checkSum (optional, can be verified on the server to verify successful transfer)
+* sequenceId (optional, ca, be assigned if not used yet in the given item)
+
```

Comment:
I really like the idea of working with the Content-Length and Content-MD5 header. This will indeed be much easier to support for REST clients. I've updated the REST contract accordingly
I noticed the wording was not clear, and have replaced the title `The bitstream metadata can contain:` with `The bitstream properties can contain:` so it's clear this is not part of the metadata section